.github/workflows/release.yml

```diff
@@ -35,8 +35,6 @@ jobs:
         run: npm run test
       - name: Coverage Report
         uses: codecov/codecov-action@v1
-      - name: Preview Changed Packages
-        run: npm run changed
```

Comment:
I previously removed this from ci.yml, but realized we should also remove it from release.yml since `lerna publish` is already going to only publish changed packages, there's no needed to fail CI if no packages have changed (as is the case in PRs like this one).